canbench-rs/src/lib.rs

```diff
@@ -488,12 +488,15 @@ pub struct BenchResult {
 /// A benchmark measurement containing various stats.
 #[derive(Debug, PartialEq, Serialize, Deserialize, CandidType, Clone, Default)]
 pub struct Measurement {
-    #[serde(default)]
+    /// Instruction counter at the start of measurement.
+    /// Not serialized, because it is not supposed to be compared to other measurements.
+    /// Used internally to correctly calculate instructions of overlapping or nested scopes.
+    #[serde(skip)]
     start_instructions: u64,
 
     /// The number of calls to the function or scope.
     #[serde(default)]
```

Comment:
Maybe this can be dropped to avoid confusion?

Reply:
Do you mean to switch to Option<u64> instead of skipping?
If so then this field will be present in the candid_result.yml files and may introduce unwanted noise.

Or do you mean to reduce the explanation in the comment?

Reply:
ah, thank you! I misunderstood to which line your comment was attached (immediately thought about the colored part).

Reply:
I reverted using extra wire-type. I'd like to keep `#[serde(default)]` here in case it's fixed in the future.